GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6_wfd.yaml

```
@@ -0,0 +1,2 @@
+alias: dc2_object_run2.2i_dr6_wfd_v1
+
```

Show this entry in the default view.
```suggestion
include_in_default_catalog_list: true
```